src/Libraries/RevitNodes/Elements/Views/Legend.cs

```diff
@@ -0,0 +1,86 @@
+﻿using DynamoServices;
+
+namespace Revit.Elements.Views
+{
+    /// <summary>
+    /// A Revit Legend View Wrapper
+    /// </summary>
+    [RegisterForTrace]
+    public class Legend : View
+    {
+        #region Internal properties
+
+        /// <summary>
+        /// An internal handle on the Revit element
+        /// </summary>
+        internal Autodesk.Revit.DB.View InternalLegend
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        public override Autodesk.Revit.DB.Element InternalElement => InternalLegend;
```

Comment:
c# 6 - honestly, after going back and forth about this, I am not sure if this will cause any problems anymore :)

Reply:
Well, let's take it out and not risk it. I will clean that up.